app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -174,42 +200,41 @@ span.preview_text_element_name {
   }
 
   .autocomplete_tag_list {
-    background-color: #e5dcca;
-    margin: 0 -8px 1px -8px;
-    padding: 2*$default-padding;
-    padding-left: 38px;
+    margin: 0 -8px 0 -8px;
+    padding: $default-padding 2*$default-padding;
 
     label {
-      top: 21px;
-      left: 8px;
       font-size: 10px;
-      z-index: 1;
     }
 
     .select2-container {
       width: 100%;
 
       .select2-choices {
-        background-color: #F1EAD9;
-
-        .select2-search-choice {
-          background-color: #e5dcca;
-        }
+        width: 100%;
+        margin-bottom: 0;
+        margin-top: 8px;
       }
     }
   }
-}
 
-.element_head, .element_foot {
-  position: relative;
+  hr {
+    height: 0;
+    width: 100%;
+    margin: 4px 0 12px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `4px 0 12px` instead of `4px 0 12px 0`